api/client.js

```diff
@@ -470,13 +471,32 @@ class Connection {
 
 
 /**
-  Convert ThisString to thisString.
+  Convert ThisString to thisString and THATString to thatString.
 
-  @param {String} string A StringLikeThis.
+  @param {String} text A StringLikeThis.
   @returns {String} A stringLikeThis.
 */
-function uncapitalize(string) {
-  return string.charAt(0).toLowerCase() + string.slice(1);
+function uncapitalize(text) {
```

Comment:
Are there tests for this?

Reply:
There are tests for the docs and js generation that indirectly cover this. Nothing internal is tested directly.